WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCouponViewModel.swift

```diff
@@ -49,10 +66,31 @@ final class AddEditCouponViewModel: ObservableObject {
         coupon = existingCoupon
         editingOption = .editing
         discountType = existingCoupon.discountType
+
+        // Populate fields
+        amountField = existingCoupon.amount
+        codeField = existingCoupon.code
     }
 
     private enum EditingOption {
         case creation
         case editing
     }
 }
+
+// MARK: - Constants
+//
+private extension AddEditCouponViewModel {
+
+    enum Localization {
+        static let amountPercent = NSLocalizedString("Amount (%)",
+                                                     comment: "Text field Amount in percentage in the view for adding or editing a coupon.")
+        static let amountFixedDiscount = NSLocalizedString("Amount (fixed discount)",
```

Comment:
I'm thinking the content between the bracket should be the currency symbol or currency code, e.g "Amount ($)" or "Amount (USD)" so that it's clearer for users when entering the amount?

Reply:
It makes totally sense de7eaeb66fef14f46740b1695e727a1e641628c5